docs/schema/kubernetes.md

```diff
@@ -0,0 +1,117 @@
+
+# Cartography - Kubernetes Schema
+
+## KubernetesCluster
+Representation of a [Kubernetes Cluster.](https://kubernetes.io/docs/concepts/overview/what-is-kubernetes/)
+
+| Field | Description |
+|-------|-------------|
+| firstseen | Timestamp of when a sync job first discovered this node |
+| lastupdated | Timestamp of the last time the node was updated |
+| id | Identifier for the cluster i.e. UID of `kube-system` namespace |
+| name | Name assigned to the cluster which is derived from kubeconfig context |
+
+### Relationships
+- KubernetesCluster has KubernetesNamespaces.
+    ```
+    (KubernetesCluster)-[HAS_NAMESPACE]->(KubernetesNamespace)
+    ```
+
+- KubernetesCluster can have KubernetesPods.
+    ```
+    (KubernetesCluster)-[HAS_POD]->(KubernetesPod)
+    ```
+
+## KubernetesNamespace
+Representation of a [Kubernetes Namespace.](https://kubernetes.io/docs/concepts/overview/working-with-objects/namespaces/)
+
+| Field | Description |
+|-------|-------------|
+| firstseen | Timestamp of when a sync job first discovered this node |
+| lastupdated | Timestamp of the last time the node was updated |
+| id | UID of the kubernetes namespace |
+| name | Name of the kubernetes namespace |
+| created_at | Timestamp of the creation time of the kubernetes namespace |
+| deleted_at | Timestamp of the deletion time of the kubernetes namespace |
+
+### Relationships
+- KubernetesNamespace can have KubernetesPods.
+    ```
+    (KubernetesNamespace)-[HAS_POD]->(KubernetesPod)
+    ```
+
+- KubernetesNamespace can have KubernetesServices.
+    ```
+    (KubernetesNamespace)-[HAS_SERVICE]->(KubernetesService)
+    ```
+
+## KubernetesPod
+Representation of a [Kubernetes Pod.](https://kubernetes.io/docs/concepts/workloads/pods/)
+
+| Field | Description |
+|-------|-------------|
+| firstseen | Timestamp of when a sync job first discovered this node |
+| lastupdated | Timestamp of the last time the node was updated |
+| id | UID of the kubernetes pod |
+| name | Name of the kubernetes pod |
+| created_at | Timestamp of the creation time of the kubernetes pod |
+| deleted_at | Timestamp of the deletion time of the kubernetes pod |
+
+### Relationships
+- KubernetesPod has KubernetesContainers.
+    ```
+    (KubernetesPod)-[HAS_CONTAINER]->(KubernetesContainer)
+    ```
+
+## KubernetesContainer
+Representation of a [Kubernetes Container.](https://kubernetes.io/docs/concepts/workloads/pods/#how-pods-manage-multiple-containers)
+
+| Field | Description |
+|-------|-------------|
+| firstseen | Timestamp of when a sync job first discovered this node |
+| lastupdated | Timestamp of the last time the node was updated |
+| id | Identifier for the container which is derived from the UID of pod and the name of container |
+| name | Name of the container in kubernetes pod |
+| image | Docker image used in the container |
+
+### Relationships
+- No relationships originate from KubernetesContainer node.
```

Comment:
I know the following relationship does not originate from KubernetesContainer but it is always efficient to add this in here.
`(KubernetesPod)-[HAS_CONTAINER]->(KubernetesContainer)`
Just add this line exactly as it is above.

Reply:
The idea is to just list the relationships that the resource has with other resources - not necessarily originating from them.